docs/section5/Rule5-9.md

```diff
@@ -0,0 +1,41 @@
+
+# Envelope - Rule 5-9  
+
+**Rule ID:** 5-9  
+**Rule Description:** Below-grade wall C-factor must be the same in the proposed model as in the user model  
+**Appendix G Section:** Section G3.1-5(a) Building Envelope Modeling Requirements for the Proposed building  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:**
+
+  - get_surface_conditioning_category()
+  - get_opaque_surface_type()
+  - match_data_element()
+
+## Rule Logic:  
+
+- Get surface conditioning category dictionary for P_RMR: `scc_dictionary_p = get_surface_conditioning_category(P_RMR)`  
+
+- For each building segment in the Proposed model: `for building_segment_p in P_RMR.building.building_segments:`  
+
+  - For each thermal_block in building segment: `for thermal_block_p in building_segment_p.thermal_blocks:`  
+
+    - For each zone in thermal block: `for zone_p in thermal_block_p.zones:`  
+
+      - For each surface in zone: `for surface_p in zone_p.surfaces:`  
+
+        - Check if surface is below-grade wall and is regulated: `if ( get_opaque_surface_type(surface_p) == "BELOW-GRADE WALL" ) AND ( scc_dictionary_p[surface_p.id] != "UNREGULATED" ):`
+
```

Comment:
Why do we need to establish that surface is regulated? Shouldn't it just be U-RMR=P-RMR

Reply:
@xingcx this check should be for all surfaces (reg and unreg)